tox.ini

```diff
@@ -1,2 +1,12 @@
+[tox]
+envlist = py32, py33, py34
```

Comment:
I know the pypi package is *currently* unneeded for python 3.5, but I think that tox should test with it.

Python 3.6 is certain to ship with a newer version of `typing`, and people running 3.5 will want to install it then in order to use the new classes.

Also, possibly comment about the 2.7 possibility (see #160).

Also enable `pypy3` (and once #160 is fixed, `pypy` and maybe `jython` - I don't have an environment installed for jython or ironpython but *ideally* they would both be supported, though they might need completely custom versions of `typing`)